service/controller/app/key/key.go

```diff
@@ -162,6 +164,31 @@ func KubecConfigSecretNamespace(customResource v1alpha1.App) string {
 	return customResource.Spec.KubeConfig.Secret.Namespace
 }
 
+func AppLabelSelector(unique bool) labels.Selector {
+	var version string
+
+	// Currently tenant cluster app CRs always have the version label
+	// app-operator.giantswarm.io/verson: 1.0.0
+	// This hardcoding will be removed in a future release and we will then
+	// use project.Version().
+	version = project.AppTenantVersion()
+	if unique {
+		// When app-operator is deployed as a unique app it only processes
+		// control plane app CRs. These CRs always have the version label
+		// app-operator.giantswarm.io/version: 0.0.0
+		version = project.AppControlPlaneVersion()
+	}
+	// Selector to check if "app=chart-operator" and if helm-major-version is not belonging to 3.
```

Comment:
Comment needs updating.

Reply:
This is leftover from thiccc copy/paste 🤦 , I'm deleting it.